modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -303,140 +303,145 @@ const fetchChoices = (getApi: (comp: AlloyComponent) => Toolbar.ToolbarSplitButt
         )
       )));
 
-// TODO: hookup onSetup and onDestroy
-const renderSplitButton = (spec: Toolbar.ToolbarSplitButton, sharedBackstage: UiFactoryBackstageShared, btnName?: string): SketchSpec => {
-  const tooltipString = Cell<string>(spec.tooltip.getOr(''));
+const getSplitButtonApi = (component: AlloyComponent): Toolbar.ToolbarSplitButtonInstanceApi => ({
+  isEnabled: () => !Disabling.isDisabled(component),
+  setEnabled: (state: boolean) => Disabling.set(component, !state),
+  setText: (text: string) => AlloyTriggers.emitWith(component, updateMenuText, { text }),
+  setIcon: (icon: string) => AlloyTriggers.emitWith(component, updateMenuIcon, { icon }),
+  setIconFill: (_id: string, _value: string) => {},
+  isActive: () => Fun.never(),
+  setActive: (_state: boolean) => {},
```

Comment:
Why are these no longer implemented?

Reply:
mmm, I didn't notice that would cause the button to miss active state, adding back now